_config.yml

```diff
@@ -8,7 +8,9 @@ sass:
   sass_dir: assets/css
   style: :compressed
 
-# Customise atom feed settings (this is where Jekyll-Feed gets configuration information)
-title:		           "Bitcoin Core PR Review Club"
-description:	       "A weekly review club for Bitcoin Core PRs"
-author:              "John Newbery"
+# Site settings
+title:           "Bitcoin Core PR Review Club"
+description:     "A weekly review club for Bitcoin Core PRs"
+author:          "Bitcoin Core PR Review Club contributors"
+meeting_time:    "18:00 UTC on Wednesday"
+irc_location:    "#bitcoin-core-pr-reviews IRC channel on [freenode](https://webchat.freenode.net/)"
```

Comment:
I could be wrong about this but I think markdown is only parsed/rendered in `{{content}}` sections? So for example the link will not look correct if I stick this string in the top of the page in https://github.com/bitcoin-core-review-club/bitcoin-core-review-club.github.io/pull/88

Reply:
`{{content}}` works but in an .html page, markdown does not... run `make preview` and open `http://localhost:4000/16702.html` to test... but you can use CSS/HTML markup if needed. I don't think you need any here though.

Reply:
Ah yes. Thanks. I've replaced this with html and it seems to work.